test-result-summary-client/src/Build/ReleaseSummary.jsx

```diff
@@ -84,7 +84,7 @@ export default class ReleaseSummary extends Component {
                                             }
                                             //For failed tests, add links to the deep history and possible issues list
                                             failedTestSummary[buildName] +=
-                                                `${testName} => [deep history ${totalPasses}/${history.length} passed](${originUrl}/deepHistory?testId=${testId}) | ` +
+                                                `[${testName}](${originUrl}/output/test?testId=${_id}) => [deep history ${totalPasses}/${history.length} passed](${originUrl}/deepHistory?testId=${testId}) | ` +
```

Comment:
I meant `${originUrl}/output/test?id=${testId}`

Reply:
Ah, yes! Sorry. This makes more sense.